test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.default.test.ts

```diff
@@ -0,0 +1,691 @@
+                      "id": "sequenceFlow_id_0",
+                      "sourceRef": "activity_id_0",
+                      "targetRef": "targetRef_RLk"
+                  }
+              },
+              "BPMNDiagram": {
+                  "id": "BpmnDiagram_1",
+                  "BPMNPlane": {
+                      "id": "BpmnPlane_1",
+                      "BPMNShape": {
+                          "id":"shape_activity_id_0",
+                          "bpmnElement":"activity_id_0",
+                          "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+                      },
+                      "BPMNEdge": {
+                          "id": "edge_sequenceFlow_id_0",
+                          "bpmnElement": "sequenceFlow_id_0"
+                      }
+                  }
+              }
+          }
+      }`;
+
+    const model = parseJsonAndExpectOnlyEdgesAndFlowNodes(json, 1, 1);
+
+    verifyEdge(model.edges[0], {
+      edgeId: 'edge_sequenceFlow_id_0',
+      bpmnElementId: 'sequenceFlow_id_0',
+      bpmnElementName: undefined,
+      bpmnElementSourceRefId: 'activity_id_0',
+      bpmnElementTargetRefId: 'targetRef_RLk',
+      bpmnIsDefault: true,
+    });
+  });
+*/
+
+  // TODO: To uncomment when we support subProcess
+  /*
+  it('json containing one process with a sequence flow defined as default in a subProcess', () => {
+    const json = `{
+          "definitions": {
+              "process": {
+                  "id": "Process_1",
+                  "subProcess": { "id": "activity_id_0", "default": "sequenceFlow_id_0"},
+                  "sequenceFlow": {
+                      "id": "sequenceFlow_id_0",
+                      "sourceRef": "activity_id_0",
+                      "targetRef": "targetRef_RLk"
+                  }
+              },
+              "BPMNDiagram": {
+                  "id": "BpmnDiagram_1",
+                  "BPMNPlane": {
+                      "id": "BpmnPlane_1",
+                      "BPMNShape": {
+                          "id":"shape_activity_id_0",
+                          "bpmnElement":"activity_id_0",
+                          "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+                      },
+                      "BPMNEdge": {
+                          "id": "edge_sequenceFlow_id_0",
+                          "bpmnElement": "sequenceFlow_id_0"
+                      }
+                  }
+              }
+          }
+      }`;
+
+    const model = parseJsonAndExpectOnlyEdgesAndFlowNodes(json, 1, 1);
+
+    verifyEdge(model.edges[0], {
+      edgeId: 'edge_sequenceFlow_id_0',
+      bpmnElementId: 'sequenceFlow_id_0',
+      bpmnElementName: undefined,
+      bpmnElementSourceRefId: 'activity_id_0',
+      bpmnElementTargetRefId: 'targetRef_RLk',
+      bpmnIsDefault: true,
+    });
+  });
+*/
+
+  it('json containing one process with a sequence flow can not be default for parallel gateway', () => {
```

Comment:
I wanted to add a test to verify that a flownode, who can not have a **default** sequenceFlow in the BPMN specification, doesn't really have a **default** sequenceFlow.

This is why I add a **default** attribute to the parallel gateway (not possible in real):
`"parallelGateway": { "id": "gateway_id_0", "default": "sequenceFlow_id_0"},`

So, I think this test corresponding to the subject Default SequenceFlow.
Maybe, I need to rename the test.

Reply:
What I had in mind was "moving the test close to the other gateways tests related to the default sequence flows in this file" not "moving the test out the current file to put it in an already existing file" :smile_cat: 
As decided, this will be managed with the test refactoring

Reply:
I renamed the test to corresponding to what I wanted test ;)